api/integrations/codebase/zulip_codebase_mirror

```diff
@@ -77,7 +77,7 @@ def make_api_call(path):
     # type: (str) -> Optional[List[Dict[str, Any]]]
     response = requests.get("https://api3.codebasehq.com/%s" % (path,),
                             auth=(config.CODEBASE_API_USERNAME, config.CODEBASE_API_KEY),
-                            params={'raw': True},
+                            params={'raw': 'True'},
```

Comment:
This is the relevant documentation:
http://docs.python-requests.org/en/master/user/quickstart/#passing-parameters-in-urls

This doesn't say whether values in the params dict must be a string or not, so I'm not sure whether it's correct to use a bool. By experimenting a bit it seems that requests stringifies values (except if it's a list or None) so the effect of both `True` and `'True'` is the same.

Since the docs don't clearly mention how non-string items will be handled, I would categorize it as undefined behavior and so I think it's more correct to use `'True'` instead of `True`.

It could be true that stringifying is the intended behavior and that's not meant to change. In that case, the typeshed stubs should be changed. In fact, it was tried to fix stubs in python/typeshed#844 and python/typeshed#849 but they forgot to include `bool`.

If you think using `'True'` is more appropriate, you can pick this commit. If you think we should continue to use `True` then leave it out. Regardless of that, I'll open a PR to add `bool` to typeshed stubs and discuss this further with typeshed maintainers.